lib/cuda/CMakeLists.txt

```diff
@@ -2,21 +2,23 @@ project(kotekan_cuda)
 
 add_library(
     kotekan_cuda
+    # Core
     cudaCommand.cpp
     cudaDeviceInterface.cpp
     cudaEventContainer.cpp
     cudaProcess.cpp
-    # Copy-in & general-purpose:
+    # Copy & general-purpose:
     cudaInputData.cpp
     cudaOutputData.cpp
     cudaOutputDataZero.cpp
-    # Simple test
-    cudaCorrelatorKernel.cu)
+    # Kernels
+    cudaCorrelatorAstron.cpp
+    cudaShuffleAstron.cu)
 
 target_link_libraries(
     kotekan_cuda
     PRIVATE libexternal kotekan_libs -lstdc++
-    INTERFACE ${CUDA_LIBRARIES})
+    INTERFACE ${CUDA_LIBRARIES} cuda nvrtc)
```

Comment:
It turns out we do...  We need both for low level memory operations and run-time compiling respectively.   Annoyingly `cmake` is also in the middle of transitioning how CUDA libs are included, but it requires a newer version of cmake than on a lot of our systems.  The new system just needs to enable the language and it auto links the core libraries. These two items move to `CUDA::nvrtc` and `CUDA::cuda_driver` which get included with a new FindCUDAToolKit() macro, but would still need to be included.    This code seems to be maximally supported, if a bit clunky since it doesn't go through `find()` or another search macro.